dali/test/python/test_operator_roi_random_crop.py

```diff
@@ -38,7 +52,7 @@ def check_roi_random_crop(ndim=2, max_batch_size=16,
         in_shape = dali.fn.shapes(shape_like_in, dtype=types.INT32)
 
         crop_shape =  [(crop_min_extent + crop_max_extent) // 2] * ndim if random.choice([True, False]) \
-            else  fn.random.uniform(range=(crop_min_extent, crop_max_extent + 1), 
+            else  fn.random.uniform(range=(crop_min_extent, crop_max_extent + 1),
```

Comment:
please trim the whitespaces after `else`

Reply:
Oh, there is double space. I don't want to rerun CI and this will touch the code.